asus/zephyrus/ga402/default.nix

```diff
@@ -0,0 +1,20 @@
+{ ... }:
+
+{
+  imports = [
+    ../../../common/cpu/amd
+    ../../../common/cpu/amd/pstate.nix
+    ../../../common/gpu/amd
+    ../../../common/pc/laptop
+    ../../../common/pc/laptop/acpi_call.nix
```

Comment:
Why is this necessary?

Reply:
I applied them because the options they contains see to be applicable to the platform (especially for the battery)

Reply:
The `acpi_call` module is only really needed for old hybrid Nvidia setups and can definitely be removed. `common/pc/laptop` can probably be left in as a no-op in case generic laptop configuration is ever desired.

Reply:
amd* are still useful right ?

Reply:
Yes.

Reply:
ok removed acpi_call module